BuildingSync.xsd

```diff
@@ -8,7 +8,9 @@
       <p>This document details the deprecation policy for the BuildingSync Schema.</p>
       <h2>Deprecating schema elements</h2>
       <p>Since the development of the BuildingSync Schema is application-driven, the schema document itself has evolved over time to reflect the evolving understanding of the problem space.</p>
-      <p><b>Rule #1: Schema elements may only be removed by incrementing the major version of the schema document.</b></p>
+      <p>
```

Comment:
Please revert changes to the `<p>` element.

Reply:
done